apps/devhub/tests/test_views.py

```diff
@@ -2539,9 +2539,9 @@ def xpi(self):
                     'rb')
 
     @mock.patch.object(waffle, 'flag_is_active')
+    @mock.patch('devhub.tasks.validate')
```

Comment:
Heh. And, since I clearly haven't rambled enough:

I had to make these changes since this code was previously simulating the delayed validation behavior by having `run_validator` return an empty string (which is the expected value for a pending task), but my new code doesn't except return values which aren't valid JSON, which caused these tests to fail.